From 3f31dd80ebbfce8829f50f3e78c417501bad96a6 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Fri, 22 Nov 2013 15:09:39 +0100
Subject: [PATCH 3/5] CMakeLists: Remove webos_machine_impl_dep call

* we're not interested in different behavior for emulator or desktop build
  calling this causes WEBOS_TARGET_MACHINE_IMPL to be set to simulator
  and webOS/LegacyDefines.cmake to define TARGET_DESKTOP

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ba01475..d867957 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,7 +54,6 @@ include_directories(${PMLOGLIB_INCLUDE_DIRS})
 link_directories(${PMLOGLIB_LIBRARY_DIRS})
 webos_add_compiler_flags(ALL ${PMLOGLIB_CFLAGS_OTHER})
 
-webos_machine_impl_dep()
 include(webOS/LegacyDefines)
 
 webos_include_install_paths()
-- 
2.41.0.windows.3

